model/security/patch_action_group/structures.smithy

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure PatchActionGroup_Input {
+    @required
+    @httpLabel
+    action_group: String
+    @httpPayload
+    content: PatchActionGroupParams
```

Comment:
Patch action group takes a list directly so this should be of type `PatchOperationList`